website/content/docs/internals/index.mdx

```diff
@@ -1,12 +1,11 @@
 ---
 layout: docs
-page_title: Internals
+page_title: Internals Overview
 description: >-
-  This section covers some of the internals of Consul, such as the architecture,
-  consensus and gossip protocols, and security model.
+Consul is a complex system that relies on a number of protocols and concepts to work. Learn about Consul's internal properties and how it works under the hood.
```

Comment:
I don't think we want to focus on complexity. Ideally, that's abstracted most of the time. The goal is to give more advanced knowledge where it's needed to promote better/more advanced usage.
```suggestion
To enhance your mental model of how to use, troubleshoot, and secure Consul, learn more about Consul's internal properties and how it works under the hood.
```